frontend/src/components/candidateCard.js

```diff
@@ -56,30 +56,34 @@ class CandidateCardComponent extends Component {
             </Row>
             <Row>
               <Col md={12}>
-                {candidate.status && candidate.status === 'Accepted' && (
-                  // Replaced the span with Badge for easier, more consistent styling
-                  <h5>
-                    <Badge color="success">Accepted</Badge>
-                  </h5>
-                )}
+                {candidate.status &&
```

Comment:
This seems overly boilerplate-d.. Is there any situation where `candidate.status` is not set?

Reply:
oh these are not even real changes lol, yeah please fix the version of `prettier` and re-request review!